dora/shell/src/main/java/alluxio/cli/fsadmin/report/SummaryCommand.java

```diff
@@ -68,121 +58,31 @@ public SummaryCommand(MetaMasterClient metaMasterClient,
+                    MasterInfoField.UP_TIME_MS, MasterInfoField.VERSION,
+                    MasterInfoField.SAFE_MODE, MasterInfoField.ZOOKEEPER_ADDRESSES,
+                    MasterInfoField.RAFT_JOURNAL, MasterInfoField.RAFT_ADDRESSES,
+                    MasterInfoField.MASTER_VERSION));
     MasterInfo masterInfo = mMetaMasterClient.getMasterInfo(masterInfoFilter);
 
-    print("Master Address: " + masterInfo.getLeaderMasterAddress());
-    print("Web Port: " + masterInfo.getWebPort());
-    print("Rpc Port: " + masterInfo.getRpcPort());
-    print("Started: " + CommonUtils.convertMsToDate(masterInfo.getStartTimeMs(),
-        mDateFormatPattern));
-    print("Uptime: " + CommonUtils.convertMsToClockTime(masterInfo.getUpTimeMs()));
-    print("Version: " + masterInfo.getVersion());
-    print("Safe Mode: " + masterInfo.getSafeMode());
-
-    List<String> zookeeperAddresses = masterInfo.getZookeeperAddressesList();
-    if (zookeeperAddresses == null || zookeeperAddresses.isEmpty()) {
-      print("Zookeeper Enabled: false");
-    } else {
-      print("Zookeeper Enabled: true");
-      print("Zookeeper Addresses: ");
-      mIndentationLevel++;
-      for (String zkAddress : zookeeperAddresses) {
-        print(zkAddress);
-      }
-      mIndentationLevel--;
-    }
-
-    if (masterInfo.getRaftJournal()) {
-      print("Raft-based Journal: true");
-      print("Raft Journal Addresses: ");
-      mIndentationLevel++;
-      for (String raftAddress : masterInfo.getRaftAddressList()) {
-        print(raftAddress);
-      }
-      mIndentationLevel--;
-    } else {
-      print("Raft-based Journal: false");
-    }
-    String formatString = "%-32s %-8s %-32s";
-    print(String.format(formatString, "Master Address", "State", "Version"));
-    for (MasterVersion masterVersion: masterInfo.getMasterVersionsList()) {
-      NetAddress address = masterVersion.getAddresses();
-      print(String.format(formatString,
-              address.getHost() + ":" + address.getRpcPort(),
-          masterVersion.getState(),
-          masterVersion.getVersion()));
-    }
-  }
-
-  /**
-   * Prints Alluxio block master information.
-   */
-  private void printBlockMasterInfo() throws IOException {
     Set<BlockMasterInfoField> blockMasterInfoFilter = new HashSet<>(Arrays
-        .asList(BlockMasterInfoField.LIVE_WORKER_NUM, BlockMasterInfoField.LOST_WORKER_NUM,
-            BlockMasterInfoField.CAPACITY_BYTES, BlockMasterInfoField.USED_BYTES,
-            BlockMasterInfoField.FREE_BYTES, BlockMasterInfoField.CAPACITY_BYTES_ON_TIERS,
-            BlockMasterInfoField.USED_BYTES_ON_TIERS));
+            .asList(BlockMasterInfoField.LIVE_WORKER_NUM, BlockMasterInfoField.LOST_WORKER_NUM,
+                    BlockMasterInfoField.CAPACITY_BYTES, BlockMasterInfoField.USED_BYTES,
+                    BlockMasterInfoField.FREE_BYTES, BlockMasterInfoField.CAPACITY_BYTES_ON_TIERS,
+                    BlockMasterInfoField.USED_BYTES_ON_TIERS));
```

Comment:
the original indentation is better, restore that here

Reply:
Strangely, my IDE indents too many spaces in this scenario. I will check the reason